### ISSUE TITLE
--dev --inspect command line option doesnt work

### ISSUE THREAD

**Issue by lihaibh** [cypress-io/cypress#23058]
### Current behavior

when i try to run the following command:
![CleanShot 2022-08-02 at 12 36 17](https://user-images.githubusercontent.com/4681233/182343461-16d4cd03-3c23-48b1-988a-dba8704fa2d8.png)

it shows me that command not found, although it listed as one of the options.

### Desired behavior

i would like to debug tasks through VSCode IDE, I want to open port for remote debugging and attach my IDE to it, in order to debug code that cypress run in the backend.
so my first approach was just running:
```
node --inspect=4000 ./node_modules/.bin/cypress open --e2e --browser chrome
```
and it worked. However, cypress is also a watcher for files, it re-runs itself when i change code, so after i change some lines of code i need to re-run the command again.
Cypress suppose to open the remote debugging option and there is an option, but it doesn't work.

### Test code to reproduce

just run:
```
./node_modules/.bin/cypress open --dev --inspect=4000 --e2e --browser chrome
```

### Cypress Version

10.3.0

**Comment by lihaibh** [comment]
any update?

**Comment by rockindahizzy** [comment]
@lihaibh thanks for reporting the issue. Thanks for providing the steps to reproduce the error. I will route this to the correct team to prioritize!

**Comment by emilyrohrbough** [comment]
@lihaibh FYI `--dev` and `--insepct` and `--inspect-brk` are CLI flags for developing Cypress, not necessarily for debugging it. Related to #21320.

**Comment by lihaibh** [comment]
@emilyrohrbough yeah i understood that from the code, its looking for some paths that i dont have. But anyway there should be an option to remote debug backend code. If i write a cypress task, i want to be able to debug it, as it is not running in the browser, i need to open remote debugging somehow and attach it to my IDE.

i would like also --inspect-brk to understand what happens during bootstrap, like code in support file.
